patches/minecraft/net/minecraft/client/server/LanServerPinger.java.patch

```diff
@@ -18,3 +22,11 @@
              DatagramPacket datagrampacket = new DatagramPacket(abyte, abyte.length, inetaddress, 4445);
              this.f_120104_.send(datagrampacket);
           } catch (IOException ioexception) {
+@@ -74,6 +_,7 @@
+       }
+    }
+ 
++   @org.jetbrains.annotations.Nullable
```

Comment:
? What is the purpose of patching in this annotation?

Reply:
Fixes some warnings regarding the method returning null from a method with `@MethodsReturnNonnullByDefault`

Reply:
But it doesn't look like this PR is even calling that method from anywhere, and even so it probably isn't worth making a patch that has to be maintained just to annotate it as nullable when it only has the benefit of warnings given there are so many places in vanilla that can return null but `@MethodsReturnNonnullByDefault` makes IDEs display warnings

Reply:
The method very obviously returns null and the fix is a simple one-liner to reduce warning spam